src/components/atoms/book-tile.astro

```diff
@@ -10,14 +10,28 @@ export interface Props extends astroHTML.JSX.HTMLAttributes {
 
 const { value, ...props } = Astro.props;
 const BookTile = 'a';
+
+const imageSizes = {
```

Comment:
Wielkość zdjęcia każego elementu z favorites powinna być elastyczna, żeby fajnie działała nam tam responsywność. To rodzic (grid) powinien decydować ile szerokości jest dostępne dla każdego elementu, a zdjęcie powinno jedynie dbać o zachowanie proporcji.